### PR TITLE
Fix casing in DatabaseExplorer.get_column_names

### PR DESCRIPTION
**Pull request by Jasopaum**

### CODE REVIEW

**Review by elsiehoffet-94 — DISMISSED** [review]
Thanks 🚀

File: pagai/services/database_explorer.py [hunk 1]
```diff
@@ -81,36 +112,50 @@ def get_column(self, column: str, table: str, schema: str = None):
...
 
-    def explore(self, table: str, limit, schema=None, filters=[]):
+    def explore(self, table_name: str, schema: str, limit: int, filters=[]):
         """
         Returns the first rows of a table alongside the column names.
         """
+        self.check_connection_exists()
+
         try:
+            columns_names = self.get_column_names(table_name, schema=schema)
```
Comment (simonvadee) [inline pagai/services/database_explorer.py:145]: why is this a variable and not the result of `self.get_table_rows(...)` ?

File: pagai/views.py [hunk 2]
```diff
@@ -11,7 +11,8 @@
 # "Allow-Control-Allow-Origin" HTTP header
 CORS(api)
 
-engines = dict()
+# Instantiate a DatabaseExplorer
+explorer = DatabaseExplorer()
```
Comment (simonvadee) [inline pagai/views.py:15]: better

**Review by simonvadee — COMMENTED** [review]
have you tested all the possible combinations for casing ?
eg: lowercase and uppercase schema for each db model ?
maybe that's complex in term of deployment setup for tests

File: pagai/services/database_explorer.py (hunk 1 above)
Comment (Jasopaum) [reply pagai/services/database_explorer.py]: I changed a bit how things work here

**Comment by Jasopaum** [comment]
> have you tested all the possible combinations for casing ?
> eg: lowercase and uppercase schema for each db model ?
> maybe that's complex in term of deployment setup for tests

I just did 🙂

File: pagai/services/database_explorer.py (hunk 1 above)
Comment (simonvadee) [reply pagai/services/database_explorer.py]: better :+1:

**Review by simonvadee — APPROVED** [review]
I hope we won't have to touch this case-sensitive db stuff for a hundred years now :pray:

**Pull request merged**